tests/performance/ecommerce_hybrid_search/common/run_vespa_benchmarks.py

```diff
@@ -0,0 +1,285 @@
+        if response.status_code == 200:
+            with open(file_path, 'wb') as f:
+                for chunk in response.iter_content(chunk_size=1024):
+                    progress_bar.update(len(chunk))
+                    f.write(chunk)
+            progress_bar.close()
+        else:
+            print(f"Failed to download {url}. Status code: {response.status_code}")
+    except Exception as e:
+        print(f"Failed to download {url} to {file_path}")
+        print(e)
+
+# Download files if they do not exist
+for query_mode, query_file in query_files.items():
+    if not os.path.exists(query_file):
+        print(f"Downloading {query_file}")
+        download_file(f"{download_base_url}/{pathlib.Path(query_file).name}", query_file)
+
+# Download feed file if it does not exist
+if not os.path.exists(feed_file_vespa):
+    download_file(f"{download_base_url}/vespa_feed-{run_size.value}.json.zst", feed_file_vespa)
+
+# %% [markdown]
+# # Vespa
+# 
+
+# %% [markdown]
+# ## Running the Vespa docker container
+# 
+# Verify that docker/podman engine is running
+# 
+# Recommended resource configuration:
+# 
+# - 8 CPUs
+# - 25 GB RAM
+# - 100 GB+ disk space
+# 
+
+# %%
+import docker
+
+client = docker.from_env()
+docker_network = "my_network"
+container_name = "vespa-ecommerce"
+# Create a network if it does not exist
+try:
+    client.networks.get(docker_network)
+    print(f"Network {docker_network} already exists")
+except docker.errors.NotFound:
+    client.networks.create(docker_network, driver="bridge")
+    print(f"Network {docker_network} created")
+
+container = client.containers.run(
+    "vespaengine/vespa",
+    detach=True,
+    ports={"8080/tcp": 8080, "19071/tcp": 19071},
+    network=docker_network,
+    name=container_name,
+)
+# Wait until container is ready
+container.reload()
+
+# %% [markdown]
+# ## Deploy the Vespa application
+# 
+
+# %%
+from vespa.deployment import VespaDocker
+
+app_package_path = "../app/"
+
+vespa_docker = VespaDocker(container=container, port=8080)
+app_name = "ecommerce"
+
+app = vespa_docker.deploy_from_disk(
+    application_name=app_name, application_root=app_package_path
+)
+
+# %% [markdown]
+# ## Feed data to Vespa
+# 
+
+# %%
+import zstandard as zstd
+import json
+import os
+
+def unzstd(file_path, remove_decompressed_file=True):
+    with open(file_path, 'rb') as compressed:
+        decomp = zstd.ZstdDecompressor()
+        with open(file_path.replace(".zst", ""), 'wb') as destination:
+            decomp.copy_stream(compressed, destination)
+    if remove_decompressed_file:
+        os.remove(file_path)
+    return file_path.replace(".zst", "")
+
+if not os.path.exists(feed_file_vespa.replace(".zst", "")):
+    unzstd(feed_file_vespa)
+
+# %%
+import subprocess
+
+# Use vespa CLI (vespa feed --progress 10) to feed data
+set_target_command = "vespa config set target local"
+feed_command = f"vespa feed --progress 10 {feed_file_vespa.replace('.zst', '')}"
```

Comment:
Ideally we should use `vespa-feed-client` to be on par with the performance test, but we can keep this for now.